fplog/fplog_test.cpp

```diff
@@ -2,6 +2,9 @@
 #include <libjson/libjson.h>
 #include "fplog.h"
 #include "utils.h"
+#include <chrono>
+#include <conio.h>
```

Comment:
Do you really need conio and windows headers here? If you do then ok but if code can be compiled without those, please remove.